EIPS/eip-3675.md

```diff
@@ -63,6 +63,12 @@ The `TERMINAL_TOTAL_DIFFICULTY` parameter is a part of client software configura
 PoW blocks that are descendants of any terminal PoW block **MUST NOT** be imported. This implies that a terminal PoW block will be the last PoW block in the canonical chain.
 
 
+### Constants
+
+| Name | Value |
+|-|-|
+| **`MAX_BYTES_PER_EXTRA_DATA`** | `32` |
```

Comment:
```suggestion
| **`MAX_EXTRA_DATA_BYTES`** | `32` |
```
Maybe update to this to be consistent with the consensus spec?

Reply:
no need for "PER" here. I was mistaken. When it is the length of a single list in a block, we don't use "PER". As in `MAX_ATTESTATIONS`. "PER" is used more for conversions

Let's use @lightclient's suggestion here